exasol/toolbox/release/__init__.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import annotations
+
+import subprocess
+from dataclasses import dataclass
+from datetime import datetime
+from functools import total_ordering
+from inspect import cleandoc
+from pathlib import Path
+from shutil import which
+
+from exasol.toolbox.error import ToolboxError
+
+
+def _index_or(container, index, default):
+    try:
+        return container[index]
+    except IndexError:
+        return default
+
+
+@total_ordering
```

Comment:
What does this mean?

Reply:
TL;DR: Implements all comparison operators based on `le` and `eq`.

[Docs: total_ordering](https://docs.python.org/3/library/functools.html#functools.total_ordering)